### PR TITLE
Feat: add ui container and api/ui options in helm chart

### PR DESCRIPTION
**Pull request by hongchaodeng**
ref: https://github.com/oam-dev/kubevela/issues/2801

- [x] api+ui 处于同一个deployment.
- [x] api+ui 默认部署，但可以配置disable。
- [x] api 默认使用kubeapi数据库

### CODE REVIEW

File: charts/vela-core/values.yaml [hunk 1]
```diff
@@ -110,12 +106,18 @@ dependCheckWait: 30s
...
-apiServer:
-  enabled: true
-  port: 8000
+
+velaux:
   replicaCount: 1
-  Service:
-    type: ClusterIP
+  apiServer:
+    image: oamdev/vela-apiserver:v1.1.2
```
Comment (barnettZQG) [inline charts/vela-core/values.yaml:113]:
```suggestion
    image: oamdev/vela-apiserver 

```

File: charts/vela-core/values.yaml [hunk 2]
```diff
@@ -110,12 +106,18 @@ dependCheckWait: 30s
...
+    Service:
+      type: ClusterIP
+  uiServer:
+    image: barnett/kubevela:velaux.11241
```
Comment (barnettZQG) [inline charts/vela-core/values.yaml:117]:
```suggestion
    image: oamdev/velaux
```

File: charts/vela-core/templates/kubevela-apiserver.yaml [hunk 3]
```diff
@@ -1,68 +1,74 @@
...
+          image: {{ .Values.imageRegistry }}{{ .Values.velaux.apiServer.image }}
           ports:
-            - containerPort: {{ .Values.apiServer.port }}
+            - containerPort: 8000
+        {{- end }}
+        {{- if .Values.velaux.uiServer }}
+        - name: {{ include "kubevela.fullname" . }}-ui
+          securityContext:
+          {{- toYaml .Values.securityContext | nindent 12 }}
+          image: {{ .Values.imageRegistry }}{{ .Values.velaux.uiServer.image }}
```
Comment (barnettZQG) [inline charts/vela-core/templates/kubevela-apiserver.yaml:39]:
```suggestion
          image: {{ .Values.imageRegistry }}{{ .Values.velaux.uiServer.image }}:{{ .Values.version}}
```

**Review by barnettZQG — COMMENTED** [review]
1.  core 和 minimal 差别是什么，请合并他们。
2.  明确指定版本并修改release脚本。
3.  对core 和api 容器设置默认的资源限额。

File: charts/vela-core/templates/kubevela-apiserver.yaml (hunk 3 above)
Comment (hongchaodeng) [reply charts/vela-core/templates/kubevela-apiserver.yaml:39]: 这个不一定是这样的吧，我们可以把版本写在 image 里，但得允许用户自己修改

File: charts/vela-core/templates/_helpers.tpl [hunk 4]
```diff
@@ -51,9 +51,9 @@ app.kubernetes.io/name: {{ include "kubevela.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end -}}
 
-{{- define "kubevela-apiserver.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "kubevela.name" . }}-apiserver
-app.kubernetes.io/instance: {{ .Release.Name }}-apiserver
+{{- define "kubevela-ux.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "kubevela.name" . }}-ux
+app.kubernetes.io/instance: {{ .Release.Name }}-ux
```
Comment (wonderflow) [inline charts/vela-core/templates/_helpers.tpl:56]: Do you mean the apiservier is ux only?

Comment (wonderflow) [reply charts/vela-core/templates/_helpers.tpl:56]: It's different with what we talked before

File: charts/vela-core/templates/kubevela-apiserver.yaml [hunk 5]
```diff
@@ -1,68 +1,74 @@
-{{- if .Values.apiServer.enabled -}}
+{{- if .Values.velaux -}}
```
Comment (wonderflow) [inline charts/vela-core/templates/kubevela-apiserver.yaml:1]: it's not seems necessary?

Comment (hongchaodeng) [reply charts/vela-core/templates/kubevela-apiserver.yaml:1]: velaux 组件是可选的

File: charts/vela-core/templates/_helpers.tpl (hunk 4 above)
Comment (hongchaodeng) [reply charts/vela-core/templates/_helpers.tpl:56]: The 'ux' includes both the APIServer and UIServer.
I don't want to bother with naming here. If you have suggestion I will take it.

**Comment by wonderflow** [comment]
closing as we use velaux as addon https://github.com/oam-dev/catalog/tree/master/addons/velaux

**Pull request closed without merging**